code/gbq/run.py

```diff
@@ -145,8 +145,6 @@ def _get_test_quantile_predictions(model_config, run_config,
     lgb_seeds = rng.integers(1e8, size=(model_config.num_bags, len(run_config.q_levels)))
     
     # training loop over bags
-    oob_preds_by_bag = np.empty((x_train.shape[0], model_config.num_bags, len(run_config.q_levels)))
-    oob_preds_by_bag[:] = np.nan
```

Comment:
`oob_preds_by_bag` are not used for any current model.  I was keeping this code around in case I decide to use it later, but there are no immediate plans for it and it was causing errors with `bag_frac_samples` equal to 1.